hack/manifests/backstage/Readme.md

```diff
@@ -0,0 +1,55 @@
+# Install parados on plain kubernetes
```

Comment:
what about having make target for this?

Reply:
It changed a bit, it was a draft PR, look at it now. I do not think that makes sense, but yeah.